src/throttler.guard.ts

```diff
@@ -124,6 +131,6 @@ export class ThrottlerGuard implements CanActivate {
    */
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   protected throwThrottlingException(context: ExecutionContext): void {
-    throw new ThrottlerException(this.errorMessage);
+    throw new ThrottlerException(this.throttlingMessage());
```

Comment:
The error message itself isn't `private` so any child class can already override it. Any reason to make a method to get it and override it instead of just overriding the message itself?

Reply:
I think this would make sense when we are doing i81n stuff

Reply:
I think overriding method is better as oppose overriding the property directly incase there's an extra check/validation before displaying throttling  message @jmcdo29

Reply:
Makes sense. This shouldn't be breaking, so would you like to add a `minor` changeset to the PR for a release?

Reply:
@jmcdo29 where do I add the minor changeset? CHANGELOG?

Reply:
Run `yarn changeset` and follow the wizard